lib/export.js

```diff
@@ -111,7 +111,7 @@ class ES6Exporter {
       output.push(`  // Ommitting getter/setter for field: ${toCommentName(field)}\n`)
     } else if (field instanceof ChoiceValue) {
       // NOTE: This is special-case code to handle choice-fields, which actually should be deprecated!
-      const choiceSymbol = toSymbol(field.options.filter(o => !(o instanceof TBD)).map(o => o.identifier.name).join('Or'));
+      const choiceSymbol = toSymbol(field.options.filter(o => !(o instanceof TBD)).map(o => sanitizeIdentifierName(o.identifier.name)).join('Or'));
```

Comment:
Since the logic here is just constructing the string argument to `toSymbol`, and since `toSymbol` does the sanitization on the argument, you should be able to actually _omit_ the call to `sanitizeIdentifierName` here.